templates/email/messages/berth_offer_new_fi.html

```diff
@@ -121,25 +121,12 @@ <h3>Venepaikan hinta</h3>
         </td>
         <td>{{ order.price }}&euro;</td>
     </tr>
-    {% for order_line in fixed_services %}
+    {% if optional_services %}
         <tr>
-            <td colspan="2">{{ order_line.product.get_service_display() }}:</td>
```

Comment:
Not sure why these are removed, are they no longer relevant? Why not remove the whole block or is it for formatting purposes?

Reply:
The diff presents it a bit stragne. What I did was to remove the `{% for order_line in fixed_services %}` block since we're not billing for `fixed services` separately.